### ISSUE TITLE
;;firstchoice command

### ISSUE THREAD

**Issue by TernundaHunter** [Frederikam/FredBoat#362]
Per the Suggestions channel, a command that will automatically select the first search result for a media request rather than having to select from a list of 5.

**Comment by Shredder121** [comment]
https://user-images.githubusercontent.com/4105066/31513277-2fc56f2e-af8e-11e7-8b7c-c3762f79e238.png

Okay, now for real.
How do we discern play, yt, sc, and other alisases and use those in `firstchoice`?
Maybe having a `feeling_lucky` option in the guild config instead if we can't find a way?

**Comment by TernundaHunter** [comment]
Would be more simple to just have ;;fc default to youtube searches. If ;;play is already youtube then I think the fact that ;;fc searches youtube and not soundcloud would be unnoticeable. I don't know anyone that types ;;sc <search term>, anyway. While a ``feeling_lucky`` config option is a possibility, I think that might be more work than necessary for this. 

Unless you want to make this whole thing *about* that.

**Comment by Shredder121** [comment]
> Unless you want to make this whole thing _about_ that option. Merging the searches.

Not necessarily, was just piping in with "we might be overseeing that we have more than one search"

Just doing the regular search, so YouTube, is viable IMO.
